target/linux/ath79/dts/qca9531_glinet_gl-xe300-nor-nand.dts

```diff
@@ -0,0 +1,153 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca953x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "glinet,gl-xe300-nor-nand", "qca,qca9531";
```

Comment:
So, what's that nor-nand suffix good for?

Reply:
This is the suffix written in the device tree of the official image.

May I change it?

Reply:
I think you should remove the suffix, as I read it you only support the 16 MB nor flash and not the nand. If a later version of the firmware does support the nand it will probably involve a mechanism similar to [the GL-AR300M](https://forum.openwrt.org/t/gl-ar300m-and-gl-ar750s-nand-support-ath79/48407) which appears fairly complicated.

Reply:
Hmm, running df -h on my device returns:
`
Filesystem                Size      Used Available Use% Mounted on

/dev/root                17.5M     17.5M         0 100% /rom
tmpfs                    60.1M    264.0K     59.8M   0% /tmp
/dev/ubi0_1              93.9M    108.0K     89.0M   0% /overlay
overlayfs:/overlay       93.9M    108.0K     89.0M   0% /
tmpfs                   512.0K         0    512.0K   0% /dev
`

So I think the nand flash is supported as linux reports 93.9 + 17.5 M, correct?

Reply:
You're entirely right, it's supported and used for the overlay. You can remove the suffix, I think this should work consistently without the complexities of the GL-AR300M approach, I don't know if there's a downside to always using the larger nand flash.

Reply:
Why is this resolved? Nothing has changed, we still have a mixture of names here...

Reply:
@adschm I just made some consistency on this subject, I hope it is resolved for you :)